frontend/src/pages/spotnet/dashboard/dashboard.css

```diff
@@ -123,11 +119,34 @@ body {
   padding: 1.5rem;
 }
 
-.balance-text-size {
-  font-size: 28px;
+/* Sum Information Styles */
+.sum-info {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.start-sum {
+  color: beige;
+  font-size: 21px;
+  font-weight: 500;
+}
+
+.current-sum {
+  font-size: 21px;
   font-weight: 500;
 }
 
+.current-sum .green {
+  color: green;
```

Comment:
use css variable for a color, according to design

Reply:
@KevinLatino this version could've been kept, just with a change of `red` to corresponding CSS variable